src/app/base/components/MainContentSection/MainContentSection.tsx

```diff
@@ -23,11 +24,14 @@ const MainContentSection = ({
   const { SIDEBAR, TOTAL } = COL_SIZES;
   return (
     <div {...props} id="main-content-section">
-      <div>
+      <ContentSection>
         {header ? (
-          <header aria-label="main content" className="row">
-            <Col size={12}>{header}</Col>
-          </header>
+          // <header aria-label="main content" className="row">
+          //   <Col size={12}>{header}</Col>
+          // </header>
+          <ContentSection.Header aria-label="main content">
```

Comment:
Do we need to do this? Does rendering a header without content cause any issues?

Reply:
Actually, this is unnecessary (and Header component does not exist). Composition of the header will depend on the page and it can either be Title or MainToolbar, which will be handled by a child component.

Reply:
Before All pages use such components, we should remove all instances of `.section-header` className and associated styles.

Reply:
So, after some messing around, removing `.section-header` styles pretty much breaks the header of every page that isn't using `MainToolbar`. For this reason, I'm gonna put this PR on hold and instead focus on knocking out the subtasks in [this ticket](https://warthogs.atlassian.net/browse/MAASENG-2440).